capstone/src/main/webapp/index.jsp

```diff
@@ -1,5 +1,10 @@
 <html>
```

Comment:
nit - add `<!DOCTYPE html>` at the start of this document.

Reply:
done!